NodesConstraints.Core/NodesConstraints.cs

```diff
@@ -1356,7 +1456,7 @@ private Constraint AddConstraint(bool enabled, Transform parentTransform, Transf
             return AddConstraint(enabled, parentTransform, childTransform, linkPosition, false, positionOffset, linkRotation, false, false, rotationOffset, linkScale, false, scaleOffset, alias);
         }
 
-        private Constraint AddConstraint(bool enabled, Transform parentTransform, Transform childTransform, bool linkPosition, bool mirrorPosition, Vector3 positionOffset, bool linkRotation, bool mirrorRotation, bool lookAt, Quaternion rotationOffset, bool linkScale, bool mirrorScale, Vector3 scaleOffset, string alias)
+        private Constraint AddConstraint(bool enabled, Transform parentTransform, Transform childTransform, bool linkPosition, bool mirrorPosition, Vector3 positionOffset, bool linkRotation, bool mirrorRotation, bool lookAt, Quaternion rotationOffset, bool linkScale, bool mirrorScale, Vector3 scaleOffset, string alias, float positionChangeFactor=1, float rotationChangeFactor=1, float scaleChangeFactor=1, float positionDamp=0, float rotationDamp=0, float scaleDamp=0, TransformLock positionLocks=null, TransformLock rotationLocks = null, TransformLock scaleLocks = null)
```

Comment:
In case something else is using AddConstraint it will still break when using default parameters until the dll is recompiled. It's safer to keep the old overload instead.

Reply:
The old original overload is still there though. Don't think any plugin would have updated already to use this second overload.
Or should I add a third overload that adds the changes of this PR?

Reply:
I didn't realize it's the mirror overload, it's fine in that case.